From 2312437b15ea9f312d39e7785cd04c23a293d0a6 Mon Sep 17 00:00:00 2001
Message-Id: <2312437b15ea9f312d39e7785cd04c23a293d0a6.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: "Kumar, Kiran S" <kiran.s.kumar@intel.com>
Date: Wed, 4 Feb 2015 17:33:04 +0530
Subject: [PATCH 06/95] SQUASHME! [VPG]: drm/i915: Removed an incorrect return
 while restoring the color state

SQUASHME! - This patch should be squashed into the following existing patch:

        Author: amanna <animesh.manna@intel.com>
        Date:   Fri Aug 1 19:19:08 2014 +0530
        Change-Id: I9ed5c1d10162cd193f2522e7bb867c493b5278b7
        REVERTME [VPG]: drm/i915: Pipe specific color manager change.

Issue: GMINL-5638
Change-Id: Ib1d2f971e280893621b0a031dc2a749797b6c6a9
Signed-off-by: Kumar, Kiran S <kiran.s.kumar@intel.com>
---
 drivers/gpu/drm/i915/intel_clrmgr.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_clrmgr.c b/drivers/gpu/drm/i915/intel_clrmgr.c
index e36c709..0ad70aa 100644
--- a/drivers/gpu/drm/i915/intel_clrmgr.c
+++ b/drivers/gpu/drm/i915/intel_clrmgr.c
@@ -1502,7 +1502,6 @@ bool intel_restore_clr_mgr_status(struct drm_device *dev)
 				DRM_ERROR("Color Restore: CSC failed\n");
 				return false;
 			}
-			return false;
 		}
 	}
 
-- 
1.7.9.5

